src/TraceEvent/Computers/WaitHandleWaitTimeComputer.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using Microsoft.Diagnostics.Tracing.Etlx;
+using Microsoft.Diagnostics.Tracing.Parsers.Clr;
+using Microsoft.Diagnostics.Tracing.Stacks;
+
+namespace Microsoft.Diagnostics.Tracing.Computers
+{
+    public class WaitHandleWaitTimeComputer
```

Comment:
Please apply the same comments from the `ContentionTimeComputer` to this one as well.